CBIIT/crdc-datahub-ui
# CRDCDH-1027 Delete orphan files

### Overview

Added ability to delete individual or all orphan files within Validation Results tab. Also, added an additional table pagination to the top of the table.

### Change Details (Specifics)

- Added individual deletion of orphan files within the "Issues" column of the Validation Results table. When clicked, it will delete the orphan file and refresh the table. This uses the `Submission.fileErrors` to determine orphan files
- Added bulk/all deletion of orphan files within the "Issues" column of the Validation Results table. When clicked, it will delete all orphan files and refresh the table
- Added tooltip to the delete all orphan files icon button
- Updated Validation Results table to have a second pagination section above the table headers
- Updated filter styling in Validation Results tab and migrated to use the form components select to be consistent

**Testing Steps:**
- Upload Data Files via CLI
- Validate
- Navigate to Validation Results
- Observe orphaned file(s) buttons, tooltip, as well as top table pagination. (Delete all orphaned files button will open a dialog)
- After all orphaned files are deleted, it requires an additional validation to resolve the "Error" state for `fileValidationStatus`.

### Related Ticket(s)

[CRDCDH-1027](https://tracker.nci.nih.gov/browse/CRDCDH-1027)
